modules/http/src/main/java/org/mule/module/http/internal/request/DefaultHttpRequester.java

```diff
@@ -87,8 +86,10 @@
     static final String WORKSTATION_DEBUG = "Workstation";
     static final String AUTHENTICATION_TYPE_DEBUG = "Authentication Type";
     static final String QUERY_PARAMS_DEBUG = "Query Params";
-    private static final String REMOTELY_CLOSED = "Remotely closed";
-    private static final int DEFAULT_RETRY_ATTEMPTS = 3;
+    public static String RETRY_ATTEMPTS_PROPERTY = SYSTEM_PROPERTY_PREFIX + "http.client.requestAttempts";
+    public static final String REMOTELY_CLOSED = "Remotely closed";
+    public static final int DEFAULT_RETRY_ATTEMPTS = 3;
+    private static int RETRY_ATTEMPTS ;
```

Comment:
Why is this defined as a static constant when you are setting/resetting it for each new instance? Should this be a non-static field?